doc/nrf/releases/release-notes-latest.rst

```diff
@@ -43,6 +43,10 @@ nRF9160
     * Removed datatype in all sending AT commands. If no sending data is specified, switch data mode to receive and send any arbitrary data.
     * Added a separate document page to describe the FOTA service.
 
+  * :ref:`asset_tracker_v2` application:
+
+    * Changed custom module responsible for controlling leds to CAF leds module.
```

Comment:
```suggestion
    * Changed the custom module responsible for controlling the LEDs to CAF LEDs module.
```
Is the module actually changed or perhaps renamed?

Reply:
There was custom module used only by Asset Tracker v2. Now ATv2 uses module leds module from CAF